media-video/ffmpeg/ffmpeg6-6.1.1.recipe

```diff
@@ -4,12 +4,12 @@ and stream audio and video. It includes libavcodec - the leading audio/video \
 codec library."
 HOMEPAGE="https://ffmpeg.org/"
 COPYRIGHT="2000-2003 Fabrice Bellard
-	2003-2023 the FFmpeg developers"
+	2003-2024 the FFmpeg developers"
 LICENSE="GNU LGPL v3"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
-CHECKSUM_SHA256="488c76e57dd9b3bee901f71d5c95eaf1db4a5a31fe46a28654e837144207c270"
-PATCHES="ffmpeg-$portVersion.patchset"
+CHECKSUM_SHA256="8684f4b00f94b85461884c3719382f1261f0d9eb3d59640a1f4ac0873616f968"
+PATCHES="ffmpeg-6.1.patchset"
```

Comment:
I think the patchset file should be renamed instead of changing the name here.

Reply:
We usually do it that way, but it is an unnecessary commit/change if nothing changes there (and it is within the same major/minor revision).

Reply:
Git detects that the file was renamed and keeps the history, so there is no problem with renaming the file. The recipe file is also renamed without any problem.

Reply:
Please name it ffmpeg-6.1.1.patchset

Reply:
Fixed.